osscameroon/camerdevs
# feat: change company to company_id column in salaries table

This pull request change company field by company_id in the `salaries` table

Steps to verify:
- move to the `backend` folder with `cd ./backend`
- run `make start-postrges` to run an initialized database
- run  `make run` to launch the api
- then run the command `curl -X GET localhost:7000/salaries` you should see something like this

```
[{"id":1,"title":"Recruiting Manager","salary":1624669,"seniority":"Seniority","city":"Livefish","company_id":994,"createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"id":2,"title":"Analog Circuit Design manager","salary":680067,"seniority":"Seniority","city":"Riffwire","company_id":808,"createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"}
....
]
```
